### PR TITLE
Lots of Flow work

### PR DESCRIPTION
**Pull request by davidtheclark**
Cleans up the aberrant spacing that Babel inflicted on Flow comments, and adds Flow comments to all of the utils.

### CODE REVIEW

File: lib/utils/isStandardSyntaxRule.js [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
 
 /**
- * Check whether a rule is standard
- *
- * @param {Rule} postcss rule node
- * @return {boolean} If `true`, the rule is standard
+ * Check whether a Node is a standard rule
  */
-module.exports = function (rule) {
+module.exports = function (rule/*: Object*/)/*: boolean*/ {
```
Comment (m-allanson) [inline lib/utils/isStandardSyntaxRule.js]: Should this be `postcss$rule` instead of `Object`?

**Comment by m-allanson** [comment]
Nice. One question, otherwise LGTM.

File: lib/utils/isStandardSyntaxRule.js (hunk 1 above)
Comment (davidtheclark) [inline lib/utils/isStandardSyntaxRule.js]: @m-allanson The fact that we check for `rule.params` below means it's confusing — might not actually be a rule.

Comment (m-allanson) [inline lib/utils/isStandardSyntaxRule.js]: @davidtheclark 👍

**Pull request merged**